app/services/clients/connector.client.js

```diff
@@ -435,10 +435,20 @@ ConnectorClient.prototype = {
     return response.data
   },
 
-  /**
-   *
-   * @param {Object} params
-   */
```

Comment:
This doc was on `updateRefundEmailEnabled` which will be deprecated in the future anyway when we move everything to be addressed by serviceId and accountType. I think the method signature and code are easy enough for anyone to understand that it doesn't require a doc.

Reply:
I think there's some value in it as the type of `enabled` doesn't seem immediately obvious - I assume it's a boolean but there's nothing explicit I can see to confirm that.

Reply:
Ok in that case (and since `updateRefundEmailEnabled` isn't really related to this PR) I'll undo this change and put back 
```
  /**
   *
   * @param {Object} params
   */
```
and it'll look like I haven't changed anything then.

Reply:
Ok I think there's some confusion - this javadoc is related to `updateRefundEmailEnabled`, not the new `updateRefundEmailEnabledByServiceId` on the right.